kolibri/plugins/device/assets/src/modules/wizard/actions/availableChannelsActions.js

```diff
@@ -47,7 +47,3 @@ export function getAllDriveChannels(store, drive) {
     };
   });
 }
-
-export function fetchUnlistedChannelInfo(store, channelId) {
```

Comment:
This action is still used at `kolibri/plugins/device/assets/src/views/AvailableChannelsPage/index.vue`.

Reply:
Oh damn, good catch. I must have only grepped js files.

Reply:
I updated this function to use `getRemoteChannelBundleByToken` because it seems to be expecting the return of multiple channels from the function.